assets/templates/partials/calendar/filters.tmpl

```diff
@@ -28,6 +28,32 @@
         {{ template "partials/calendar/filter/release-type" . }}
       </div>
     </div>
+    <div
+      class="ons-collapsible ons-js-collapsible ons-collapsible--accordion"
+      id="filter-release-type"
+      data-btn-close="Hide"
+      data-group="accordion"
+      {{ if .FuncIsFilterCensusPresent }}
+        data-open="true"
+      {{ else }}
+        data-open="false"
+      {{ end }}
+    >
+      <div
+        class="ons-collapsible__heading ons-js-collapsible-heading"
+        tabindex="0"
+      >
+        <div class="ons-collapsible__controls">
+          <h2 class="ons-collapsible__title">
+            {{- localise "ReleaseCalendarFilterCensus" .Language 1 -}}
```

Comment:
This is missing the latest addition to the MVP design.

![Screenshot 2022-08-05 at 10 14 29](https://user-images.githubusercontent.com/912770/183047253-975a534e-8efb-4770-92aa-41b3dff49bce.png)